src/main/java/net/dv8tion/jda/internal/entities/channel/mixin/attribute/ITopicChannelMixin.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright 2015 Austin Keener, Michael Ritter, Florian Spieß, and the JDA contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.dv8tion.jda.internal.entities.channel.mixin.attribute;
+
+import net.dv8tion.jda.internal.entities.channel.mixin.middleman.GuildChannelMixin;
+
+public interface ITopicChannelMixin<T extends ITopicChannelMixin<T>> extends GuildChannelMixin<T>
+{
+    T setTopic(String topic);
+
+    String getTopic();
```

Comment:
Why is the getter here in the mixin?

Reply:
Because there is no other interface for it, I didn't feel it necessary to add an `ITopicableChannel`